FinancialTracker/Helpers/Extensions.swift

```diff
@@ -66,30 +66,32 @@ extension Date {
     func formatDate(_ format: String) -> String {
         let dateformat = DateFormatter()
         dateformat.dateFormat = format
+        dateformat.locale = Locale(identifier: "bg_BG")
```

Comment:
It is not a great idea to hardcode the `Locale`. Better stick with what the phone provides.